airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -386,15 +480,29 @@ def start_pipeline(
             pipeline_name,
             "workflows",
             "DataPipelineWorkflow",
-            "start"
+            "start",
         )
+        runtime_args = runtime_args or {}
+        # Unfortunately making the start call to CDAP does not return a run_id to poll for state.
```

Comment:
nit: IMO this should provide context link to relevant  CDAP bug https://issues.cask.co/browse/CDAP-7641
This way once that is closed we can remember to simplify this.
```suggestion
        # Unfortunately making the start call to CDAP does not return a run_id to poll for state.
        # https://issues.cask.co/browse/CDAP-7641
```

Reply:
You can avoid using the faux run Id by making a call to the batch start pipeline endpoint - the run id will be returned in this case. For example:
TYPE: POST
URL: 'https://xxx.datafusion.googleusercontent.com/api/v3/namespaces/default/start'
BODY: 
[{"appId": "app_id", "programType": "workflow", "programId": "DataPipelineWorkflow","runtimeargs": {}}]

Batch start pipeline endpoint info:
https://docs.cdap.io/cdap/current/en/reference-manual/http-restful-api/lifecycle.html#H3293 (documentation does not currently reflect that the run Id is returned)